tests/unit/date/helper.js

```diff
@@ -0,0 +1,30 @@
+define( [
+	"jquery",
+	"globalize",
+	"globalize/date",
+	"lib/helper"
```

Comment:
Works as expected, what do you mean by "inaccessible"? I adopted this from here: https://github.com/jquery/jquery-ui/blob/master/tests/unit/dialog/helper.js#L3